class/bullet2.js

```diff
@@ -0,0 +1,42 @@
+class Bullet2 {
+    constructor(x, y) {
+        this.x = x;
+        this.y = y;
+        //circle radius
+        this.r = 2;
+        this.toRemove = false;
+    }
+
+
+    show() {
+        imageMode(CENTER)
+        image(bulletGreen, this.x, this.y)
```

Comment:
Maybe a good idea is to pass color as a parameter when we create a new bullet so we don't have to duplicate Bullet class.